Sources/SWBUtil/POSIX.swift

```diff
@@ -15,17 +15,22 @@ import SWBLibc
 public import protocol Foundation.LocalizedError
 
 #if os(Windows)
-#if canImport(System)
-import System
-#else
-import SystemPackage
-#endif
+public import Foundation
+
+private func strerror(_ code: CInt) -> String {
+    withUnsafeTemporaryAllocation(of: CChar.self, capacity: 95) {
```

Comment:
95?

Reply:
94 + 1; the strerror message is limited to 94 characters.

Reply:
Thanks, could you encode that in a comment (with a link to docs?) so it doesn't look like an arbitrary magic to future readers?